torch/_inductor/scheduler.py

```diff
@@ -377,7 +377,7 @@ def can_inplace(self, read_dep: dependencies.Dep) -> bool:
     def has_side_effects(self) -> bool:
         return False
 
-    def decide_inplace_update(self) -> None:
+    def decide_inplace_update(self, fused_nodes: Optional[Set[str]] = None) -> None:
```

Comment:
We don't need to pass this through. You can access the fused node through: `self.scheduler.name_to_fused_node`